src/System.IO.Compression.ZipFile/pkg/System.IO.Compression.ZipFile.pkgproj

```diff
@@ -1,10 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.IO.Compression.ZipFile.csproj">
```

Comment:
Removing the live ref build still worries me a little because people won't know they need to add this back when the add APIs.